static/app/views/dashboards/widgetBuilder/issueWidget/utils.tsx

```diff
@@ -27,19 +26,3 @@ export function generateIssueWidgetFieldOptions(
 
   return fieldOptions;
 }
-
-export const ISSUE_WIDGET_SORT_OPTIONS = [
-  IssueSortOptions.DATE,
-  IssueSortOptions.NEW,
-  IssueSortOptions.FREQ,
-  IssueSortOptions.TRENDS,
-  IssueSortOptions.USER,
-];
-
-export function generateIssueWidgetOrderOptions(): Array<SelectValue<string>> {
```

Comment:
Needed to update this due to the `getSortLabel` change, but it doesn't seem to be used anywhere